Automattic/pocket-casts-android
# Move localization download to Fastlane

## Description
Uses Fastlane to download localizations instead of an standalone script, per request from @ashiagr.


Fixes # 

## Testing Instructions
The easiest way to test this is by making a new, temporary lane in the `Fastfile`, containing:
```ruby
  lane :test do 
    # Download Localizations
    android_download_translations(
      res_dir: File.join('modules', 'services', 'localization', 'src', 'main', 'res'),
      glotpress_url: GLOTPRESS_APP_STRINGS_PROJECT_URL,
      locales: SUPPORTED_LOCALES,
      lint_task: 'lintRelease'
    )
  end
```

Then you can run `bundle install && bundle exec fastlane test`, which will download the translations using the release toolkit – you'll see significant diffs on the translation files (and each translation should be changed).

This PR does _not_ include the changes to the translation files at this point – it'd make things hard to review, so that's better done once this PR is approved.


## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
